nms/app/components/CustomMetrics.tsx

```diff
@@ -111,9 +111,6 @@ export default function CustomHistogram(props: Props) {
         data={{datasets: props.dataset}}
         options={{
           maintainAspectRatio: false,
-          // TODO[TS-migration is this a valid chart.js option?]
-          // @ts-ignore
-          scaleShowValues: true,
```

Comment:
I search through the code base of `react-chart-js` and `chart-js` and could not find "scaleShowValues".